README.md

```diff
@@ -15,7 +15,7 @@ Book a [1-on-1 Session](https://cal.com/taranjeetio/ec) with Taranjeet, the foun
 ## 🔧 Quick install
 
 ```bash
-pip install --upgrade embedchain
+pip install --upgrade "embedchain"
```

Comment:
Hey thanks for your contribution.
But we need double quotes where there is extra

eg: (where double quotes are needed)
```
pip install --upgrade "embedchain[poe]"
```

eg: (where double quotes are not needed)
```
pip install --upgrade embedchain
```